src/PrintProvider.jsx

```diff
@@ -83,40 +109,14 @@ export default function PrintProvider (props) {
         printableRegistry.current[key]
       )
     );
-    printableRegistry.current = {
-      ...printableRegistry.current,
+    printableRegistry.current = Object.assign({}, printableRegistry.current, {
```

Comment:
why `Object.assign({}`, again? o_O

Reply:
The build was failing because of the spread operator

![image](https://user-images.githubusercontent.com/14141039/69542183-a7d5a280-0f69-11ea-993e-e7fd43071af5.png)

Maybe we need to install some babel plugin ?

Reply:
looks like babel-preset-env is out of date,

ok, let's keep 'Object.assign' now.